roles/edpm_ovs/tasks/update.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Update openvswitch packages without stopping the service
+  become: true
+  tags:
+    - update
+    - ovs
+  ansible.builtin.shell: >
+    dnf update -y openvswitch --setopt=tsflags=noscripts
```

Comment:
please add a comment here explaining why we have to `shell` and not use a ansible module for `dnf` or `rpm`.

please add a comment explaining why we have to skip scripts. Please link to the design document in dev-docs repo here, so that a future reader has context to dive into if needed.

please document the assumption that no useful scripts are defined in the rpm spec (and that's why we think it's safe to skip all of them).